### DISCUSSION TITLE
Allow avatars to be uploaded

### DISCUSSION THREAD

**Discussion by sanny-io** [npm/feedback#461] [Website] [answered]
Really should not have to sign up for a completely different service just to use an avatar.

**Comment by MylesBorins** [comment, marked as answer]
@sanny-io Thanks for the suggestion. At this time we are not looking to move off of Gravatar for account images.

**Reply by sanny-io** [reply]
But is this something that's completely off the table or something that has such low priority it won't be implemented any time soon? I would hope it's the latter 😓

**Reply by MylesBorins** [reply]
It's the latter for sure.

But isn't something we would be able to commit to on the road map right now

**Reply by sanny-io** [reply]
Thanks, that's totally understandable.
